packages/@ama-sdk/schematics/schematics/typescript/core/schema.ts

```diff
@@ -12,4 +12,11 @@ export interface NgGenerateTypescriptSDKCoreSchematicsSchema extends JsonObject
 
   /** Path to the spec generation configuration */
   specConfigPath: string;
+
+  /**
+   * Comma separated string of options to give to the openapi-generator-cli
+   * Example: debugModels to log the full json structure used to generate models
+   * Example: debugOperations to log the full json structure used to generate operations
+   */
+  globalProperty: string;
```

Comment:
Is it missing in `schema.json` on purpose or just forgot ?

Reply:
indeed :D